packages/embedded-wallet/src/ui/slider/Slider.test.tsx

```diff
@@ -0,0 +1,129 @@
+import Enzyme, {mount, ReactWrapper} from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import React from 'react';
+import {Slider, SliderProps} from './Slider';
+import css from './Slider.module.css';
+
+Enzyme.configure({adapter: new Adapter()});
+
+type MockSlider = {
+  sliderWrapper: ReactWrapper<SliderProps>;
+  minLabelElement: ReactWrapper;
+  sliderElement: ReactWrapper;
+  maxLabelElement: ReactWrapper;
+  valueLabelElement: ReactWrapper;
+};
+
+const mockSlider = (props?: Partial<SliderProps>): MockSlider => {
+  const sliderWrapper = mount<SliderProps>(
+    <Slider min={0} max={100} step={1} unit="ETH" {...props} />
+  );
+
+  return refreshSliderFrom({sliderWrapper} as MockSlider);
+};
+
+const refreshSliderFrom = (slider: MockSlider) => {
+  const {sliderWrapper} = slider;
+
```

Comment:
Umm... you destructured sliderWrapper, then you use it directly... TOC's!!! TOC's triggered!

Reply:
Oh, evil me.

![Maleficent looks angry.](https://media.giphy.com/media/dh7lq1Pv1sZ2/giphy.gif)